package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "s47_most_searched_website_on-google",
+  "version": "1.0.0",
+  "description": "Idea title : Most googled website in 2023",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Consider adding a meaningful script under the "scripts" field instead of the placeholder error message. This could be a script for running tests or starting the application. [important]